### PR TITLE
translate binding page

### PR DESCRIPTION
**Pull request by syxunion**
As title
Fixex #33301

### CODE REVIEW

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+`apiVersion: v1`
+
+`import "k8s.io/api/core/v1"`
+
+
+## Binding {#Binding}
+<!--
+Binding ties one object to another; for example, a pod is bound to a node by a scheduler. Deprecated in 1.7, please use the bindings subresource of pods instead.
+-->
+Binding 即将一个对象与另一个对象联系起来;例如，一个 pod 被调度程序绑定到一个节点。1.7已弃用，请使用 pods 的 bindings 子资源。
```
Comment (my-git9) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:33]:
```suggestion
Binding 将一个对象与另一个对象联系起来；例如，一个 Pod 被调度程序绑定到一个节点。1.7 已弃用，请使用 Pod 的 Binding 子资源。
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
+---
+api_metadata:
+  apiVersion: "v1"
+  import: "k8s.io/api/core/v1"
+  kind: "Binding"
+content_type: "api_reference"
+description: "Binding 即将一个对象与另一个对象联系起来;例如，一个 pod 被调度程序绑定到一个节点。"
```
Comment (my-git9) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:7]:
```suggestion
description: "Binding 将一个对象与另一个对象联系起来；例如，一个 Pod 被调度程序绑定到一个节点。"
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+<!--
+  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+
+- **target** (<a href="{{< ref "../common-definitions/object-reference#ObjectReference" >}}">ObjectReference</a>), required
+
+  The target object that you want to bind to the standard object.
+-->
+  标准对象的元数据 更多信息: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
```
Comment (my-git9) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:50]:
```suggestion
  标准对象的元数据，更多信息: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+401: Unauthorized
+
+<!--
+### `create` create binding of a Pod
+
+#### HTTP Request
+
+POST /api/v1/namespaces/{namespace}/pods/{name}/binding
+-->
+### `create` 创建Pod的绑定
```
Comment (my-git9) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:153]:
```suggestion
### `create` 创建 Pod 的 Binding
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+<!--
+### `create` create a Binding
+
+#### HTTP Request
+
+POST /api/v1/namespaces/{namespace}/bindings
+
+#### Parameters
+-->
+### `创建` 创建一个 Binding
```
Comment (my-git9) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:69]:
```suggestion
### `create` 创建一个 Binding
```

Comment (syxunion) [reply content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md]: Thanks.It has been modified according to comments.

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md [hunk 6]
```diff
@@ -0,0 +1,228 @@
...
+
+<!--
+- **namespace** (*in path*): string, required
+
+  <a href="{{< ref "../common-parameters/common-parameters#namespace" >}}">namespace</a>
+
+
+- **body**: <a href="{{< ref "../cluster-resources/binding-v1#Binding" >}}">Binding</a>, required
+-->
+- **namespace** (**在路径上**): string, 必需的
```
Comment (javadoors) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:85]:
```suggestion
- **namespace** (*在路径上*): string, 必需的
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md [hunk 7]
```diff
@@ -0,0 +1,228 @@
...
+
+  <a href="{{< ref "../common-parameters/common-parameters#dryRun" >}}">dryRun</a>
+
+
+- **fieldManager** (*in query*): string
+
+  <a href="{{< ref "../common-parameters/common-parameters#fieldManager" >}}">fieldManager</a>
+
+-->
+- **dryRun** (**查询字符串**): string
```
Comment (javadoors) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:103]:
```suggestion
- **dryRun** (*查询字符串*): string
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md [hunk 8]
```diff
@@ -0,0 +1,228 @@
...
+
+- **fieldValidation** (*in query*): string
+
+  <a href="{{< ref "../common-parameters/common-parameters#fieldValidation" >}}">fieldValidation</a>
+
+- **pretty** (*in query*): string
+
+  <a href="{{< ref "../common-parameters/common-parameters#pretty" >}}">pretty</a>
+-->
+- **dryRun** (**查询字符串**): string
```
Comment (javadoors) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:203]:
```suggestion
- **dryRun** (*查询字符串*): string
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md [hunk 9]
```diff
@@ -0,0 +1,228 @@
...
+
+  <a href="{{< ref "../common-parameters/common-parameters#dryRun" >}}">dryRun</a>
+
+
+- **fieldManager** (**查询字符串**): string
```
Comment (javadoors) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:208]:
```suggestion
- **fieldManager** (*查询字符串*): string
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md [hunk 10]
```diff
@@ -0,0 +1,228 @@
...
+
+  <a href="{{< ref "../common-parameters/common-parameters#fieldManager" >}}">fieldManager</a>
+
+- **fieldValidation** (**查询字符串**): string
```
Comment (javadoors) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:212]:
```suggestion
- **fieldValidation** (*查询字符串*): string
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md [hunk 11]
```diff
@@ -0,0 +1,228 @@
...
+
+  <a href="{{< ref "../common-parameters/common-parameters#fieldValidation" >}}">fieldValidation</a>
+
+- **pretty** (**查询字符串**): string
```
Comment (javadoors) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:216]:
```suggestion
- **pretty** (*查询字符串*): string
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md [hunk 12]
```diff
@@ -0,0 +1,228 @@
...
+
+  <a href="{{< ref "../common-parameters/common-parameters#fieldValidation" >}}">fieldValidation</a>
+
+
+- **pretty** (*in query*): string
+
+  <a href="{{< ref "../common-parameters/common-parameters#pretty" >}}">pretty</a>
+-->
+
+- **fieldValidation** (**查询字符串**): string
```
Comment (javadoors) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:124]:
```suggestion
- **fieldValidation** (*查询字符串*): string
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md [hunk 13]
```diff
@@ -0,0 +1,228 @@
...
+
+  <a href="{{< ref "../common-parameters/common-parameters#fieldValidation" >}}">fieldValidation</a>
+
+
+- **pretty** (**查询字符串**): string
```
Comment (javadoors) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:129]:
```suggestion
- **pretty** (*查询字符串*): string
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md [hunk 14]
```diff
@@ -0,0 +1,228 @@
...
+
+  <a href="{{< ref "../common-parameters/common-parameters#fieldManager" >}}">fieldManager</a>
+
+-->
+- **dryRun** (**查询字符串**): string
+
+  <a href="{{< ref "../common-parameters/common-parameters#dryRun" >}}">dryRun</a>
+
+
+- **fieldManager** (**查询字符串**): string
```
Comment (javadoors) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:108]:
```suggestion
- **fieldManager** (*查询字符串*): string
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md [hunk 15]
```diff
@@ -0,0 +1,228 @@
...
+The file is auto-generated from the Go source code of the component using a generic
+[generator](https://github.com/kubernetes-sigs/reference-docs/). To learn how
+to generate the reference documentation, please read
+[Contributing to the reference documentation](/docs/contribute/generate-ref-docs/).
+To update the reference content, please follow the 
+[Contributing upstream](/docs/contribute/generate-ref-docs/contribute-upstream/)
+guide. You can file document formatting bugs against the
+[reference-docs](https://github.com/kubernetes-sigs/reference-docs/) project.
+-->
+
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:23]: The above paragraph can be safely removed.

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md [hunk 16]
```diff
@@ -0,0 +1,228 @@
...
+`apiVersion: v1`
+
+`import "k8s.io/api/core/v1"`
+
+
+## Binding {#Binding}
+<!--
+Binding ties one object to another; for example, a pod is bound to a node by a scheduler. Deprecated in 1.7, please use the bindings subresource of pods instead.
+-->
+Binding 将一个对象与另一个对象联系起来; 例如，一个 Pod 被调度程序绑定到一个节点。1.7 已弃用，请使用 Pod 的 Binding 子资源。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:33]: binding 子资源首字母必须小写。
适当折行。

```suggestion
Binding 将一个对象与另一个对象联系起来; 例如，一个 Pod 被调度程序绑定到一个节点。
已在 1.7 版本中弃用，请使用 Pod 的 binding 子资源。
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md [hunk 17]
```diff
@@ -0,0 +1,228 @@
...
+
+
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+<!--
+  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+
+- **target** (<a href="{{< ref "../common-definitions/object-reference#ObjectReference" >}}">ObjectReference</a>), required
+
+  The target object that you want to bind to the standard object.
+-->
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:49]: 不要这样注释分段。可以先注释44行并翻译，再注释 46 - 48 行并翻译。

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md [hunk 18]
```diff
@@ -0,0 +1,228 @@
...
+  标准对象的元数据, 更多信息: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:50]: 标点符号用全角字符

```suggestion
  标准的对象元数据，更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md [hunk 19]
```diff
@@ -0,0 +1,228 @@
...
+
+- **target** (<a href="{{< ref "../common-definitions/object-reference#ObjectReference" >}}">ObjectReference</a>), 必需的
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:52]:
```suggestion
- **target** (<a href="{{< ref "../common-definitions/object-reference#ObjectReference" >}}">ObjectReference</a>)，必需
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md [hunk 20]
```diff
@@ -0,0 +1,228 @@
...
+
+  要绑定到标准对象的目标对象。
+
+## Operations {#Operations}
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:56]:
```suggestion
<!--
## Operations {#Operations}
-->
## 操作   {#operations}
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md (hunk 6 above)
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:85]:
```suggestion
- **namespace** (**路径参数**)：string，必需
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md (hunk 7 above)
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:103]:
```suggestion
- **dryRun** (**查询参数**)：string
```

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md (hunk 6 above)
Comment (tengqm) [reply content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:85]: 单星号改为双星号是对的。单星号斜体很难看，不适合中文。

Comment (javadoors) [reply content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md:85]: 了解了：）

**Comment by syxunion** [comment]
Thanks. It has been modified according to review

File: content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md (hunk 7 above)
Comment (syxunion) [reply content/zh/docs/reference/kubernetes-api/cluster-resources/binding-v1.md]: Thanks. It has been modified according to review

**Pull request merged**